README.md

```diff
@@ -120,4 +120,12 @@ On Xenial
 
 On Bionic
 
-    rocker --nvidia osrf/ros:melodic-desktop-full gazebo
\ No newline at end of file
+    rocker --nvidia osrf/ros:melodic-desktop-full gazebo
+
+## Mount volumes
+
+`--mount` adds paths as docker volumes.
+The path used inside the container is the same as the path outside.
+The last path must be terminated with two slashes `--`.
+
+    rocker --mount ~/.vimrc ~/.bashrc -- ubuntu:18.04
```

Comment:
```suggestion
    rocker --mount ~/.vimrc:~/.vimrc ~/.bashrc:~/.bashrc -- ubuntu:18.04
```

Reply:
If volume args are passed unmodified then this becomes `~/.vimrc:/home/username/.vimrc ...` because `~` appears to only get expanded at the start of an argument.